tests/Http/Three/ReviewsTest.php

```diff
@@ -74,7 +122,7 @@ public function testUpdatedPostAndSignupWithReview()
 
         // Review the post.
         $northstarId = $this->faker->northstar_id;
-        $this->withAccessToken($northstarId, 'admin')->postJson('api/v3/reviews', [
+        $this->withAccessToken($northstarId, 'admin', ['activity', 'write'])->postJson('api/v3/reviews', [
```

Comment:
I'd love to make an update to Gateway's [WithOAuthTokens](https://github.com/DoSomething/gateway/blob/master/src/Testing/WithOAuthTokens.php) trait that allows it to read default scopes from a property on the class, so that we can keep using `withAdminAccessToken` here. (Or even just add `activity` to the list of scopes hard-coded into that trait.)

Reply:
@DFurnes Ah ok that makes more sense! So should I just add it [here](https://github.com/DoSomething/gateway/blob/master/src/Testing/WithOAuthTokens.php#L54) in the `scopes` array?

Reply:
Yeah, I think that'd be the easiest fix!

Reply:
great! I'll add the `activity` scope there and then keep the `write` scopes in these tests in Rogue!

Reply:
[here's](https://github.com/DoSomething/gateway/compare/scopes?expand=1) the PR :)